lib/elevator.js

```diff
@@ -100,7 +106,10 @@ exports.execute = function(command, options, callback) {
  */
 exports.executeSync = function(command, options) {
   try {
-    return child_process.execFileSync(commandBuilder.getBinaryPath(), commandBuilder.build(command, options), {
+    var binaryPath = commandBuilder.getBinaryPath();
+    debug('executing: %s', binaryPath);
```

Comment:
Would it help if these debug strings clarified whether the `execute` or `executeSync` version is being called?